addons/AddOns/Threshold/ThresholdView.cpp

```diff
@@ -49,177 +50,176 @@ ThresholdView::ThresholdView(BMessage *msg)
 		.Add(modeMenu)
 		.AddGlue()
 		.SetInsets(B_USE_SMALL_INSETS)
-		.End();
+	.End();
```

Comment:
this looked better before.

Reply:
@dsizzle seems to prefer the .End() de-indented everywhere else. The addons haven't been touched as much, but I changed the style to fit with the rest.

Reply:
dann OK :)